SDK docs Spryk section review and edits (#1354)
spryker/spryker-docs: docs/sdk/dev/spryks/fileresolver.md

template: howto-guide-template
---

The FileResolver, located in `SprykerSdk\Spryk\Model\Spryk\Builder\Resolver\FileResolverInterface`, is a core part of the file management.
It collects all the updated or created files and flushes them into the filesystem at the end of the Spryk command execution.
All new or updated files must always be added to the FileResolver.

> Normally, you shouldn't work with the file system directly through PHP functions like `file_put_contents`, `file_get_contents`, `file_exists`, and so on.

Reviewer: avoid "so on"
```suggestion
Normally, you shouldn't work with the file system directly through PHP functions like `file_put_contents`, `file_get_contents`, or `file_exists`.
```

> You must use `FileResolverInterface::hasResolved()`, `FileResolverInterface::resolve()`, `FileResolverInterface::addFile()`.

Reviewer: ```suggestion
You must use `FileResolverInterface::hasResolved()`, `FileResolverInterface::resolve()`, and `FileResolverInterface::addFile()`.
```
